src/fitnesseMain/FitNesseMain.java

```diff
@@ -1,3 +1,5 @@
+// trigger jenkins
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Comment matches to-do format '.*'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.cloudmok.org/coding_rules#rule_key=checkstyle%3ACheck_Any_Comments)